### DISCUSSION TITLE
使用本地部署，运行了main.py，但webui连接不上

### DISCUSSION THREAD

**Discussion by musearcosu** [EstrellaXD/Auto_Bangumi#629] [Q&A]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/EstrellaXD/Auto_Bangumi/releases/latest) 相同。
- [X] 我已经查阅了[排错流程](https://autobangumi.org/faq/排错流程.html)，确保提出的问题不在其中。
- [X] 我已经查阅了[已知问题](https://autobangumi.org/faq/常见问题.html)，并确认我的问题不在其中。
- [X] 我已经 [issue](https://github.com/EstrellaXD/Auto_Bangumi/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

3.1

### 问题类型

WebUI

### 问题描述

使用本地部署的方法，步骤都跟上了，但是webui都打不开。
虽然很想使用docker部署，但是作为小白尝试了许多遍依旧不行。
环境：Windows 11

**Comment by EstrellaXD** [comment]
这个问题非程序bug，已经转成 Discussion。
请提供一下运行代码之后得到的输出日志。

**Reply by musearcosu** [reply]
运行代码之后没有任何输出。。。

**Reply by AiharaMahiru** [reply]
在舍友在PC上复现了这个问题，感觉是系统环境导致的

**Reply by EstrellaXD** [reply]
Python环境是否正确

**Comment by yeechern10** [comment]
同样是windows运行，同样是webui加载不了，但我用的是docker，最后解决了
在csdn上看到说是因为docker是运行在创建的一个Linux环境下，docker中的localhost和windows10里的不一样
文章里的解决办法是用docker-machine ip default来获得linux的ip地址，然后用这个ip加上断口就能访问
于是我用git bash安装了docker-machine，但神奇的是docker-machin安装后就能连上windows自己的的ip连上webui端口了

**Comment by SSttuuddyy** [comment]
可以尝试更改config文件夹下的config.json中第五行的"webui_port":7892，改为任意一个端口号

**Comment by PokeLu** [comment]
在fnos的docker上复现了该问题（不过这个系统是不是就相当于一个带了前端的debian，我是通过命令行的docker compose部署的），config中随便改了个端口就好了。其他容器不存在该问题，很迷。
